md/forms.md

```diff
@@ -23,4 +23,19 @@ The overview consists of three main sections:
 * List of documents: it shows the content of each document used by the case.
 * Timeline: it shows in chronological order information about all the actions that have been performed in the selected case.
 
-You can see how to manage the link between process and and form in the documenation about [live update](live-update.md)
+To learn how to manage the link between process and forms, go to the [live update](live-update.md) page.
+
+### Application Theme access
+
+If your forms is viewed in an application, you will have access facilities for [the application theme](applications.md).
+
+The `Theme.css` is directly accessible by adding the following link in `index.html`: `<link href="../theme/theme.css" rel="stylesheet" />`
+This link is already inserted in the forms you design with the UI Designer.
+
+::: info 
+Theme is loaded via the `app` URL parameter. The living application layout inject this `app` URL parameter in the targeted page/form URL.
```

Comment:
I would add "but if you create your own navigation link, you will need to add it to the target form URL in order to be able to benefit from the application theme. The value of this parameter is the application URL token (as defined when creating the application)."